libgit/init.go

```diff
@@ -109,3 +110,56 @@ func Init(ctx context.Context, gitKBFSParams libkbfs.InitParams,
 
 	return ctx, config, nil
 }
+
+// KeybaseServiceCn defines methods needed to construct KeybaseService
+// and Crypto implementations.
+type keybaseServicePassthrough struct {
+	config libkbfs.Config
+}
+
+func (ksp keybaseServicePassthrough) NewKeybaseService(
+	_ libkbfs.Config, _ libkbfs.InitParams, _ libkbfs.Context,
+	_ logger.Logger) (libkbfs.KeybaseService, error) {
+	return ksp.config.KeybaseService(), nil
+}
+
+func (ksp keybaseServicePassthrough) NewCrypto(
+	_ libkbfs.Config, _ libkbfs.InitParams, _ libkbfs.Context,
+	_ logger.Logger) (libkbfs.Crypto, error) {
+	return ksp.config.Crypto(), nil
+}
+
+var _ libkbfs.KeybaseServiceCn = keybaseServicePassthrough{}
+
+func getNewConfig(
```

Comment:
I think you mean `gitNewConfig` 😆

Reply:
No, I don't think I meant that.  "git" is already implied in the library name.  If you don't like the "get", I can drop it and just name the function `newConfig()`.

Reply:
I know, I was kidding :)

Reply:
The emoji doesn't come across in email unfortunately.

Reply:
Oops sorry, I'm not well versed in emoji.  :older_man: